task/sanity-inspect-image/0.1/sanity-inspect-image.yaml

```
@@ -71,7 +71,8 @@ spec:
       if [ ! -z $(cat stderr.txt) ]; then
         echo "skopeo inspect fails, the sanity-inspect-image test meets the following error:"
         cat stderr.txt
-        HACBS_TEST_OUTPUT="$(make_result_json -r ERROR -t 'skopeo inspect meets errors')"
+        note="Task $(context.task.name) meets error: skopeo inspect meets errors"
```

Skopeo is implementation detail, we probably should say this to users.
```suggestion
        note="Task $(context.task.name) failed: encountered error(s) while inspecting the image"
```

---

Yes, thanks. But the error might be long, so let's update note to `Task $(context.task.name) failed: Encountered error(s) while inspecting the image". For detailed error, check the Tekton task log.`